### PR TITLE
Disable "Accept All" button after reply all changes.

### PR DESCRIPTION
**Pull request by victoria34**
Issue #1183

### CODE REVIEW

File: web/yo/app/scripts/controllers/genes.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ angular.module('oncokbApp')
                 DTColumnDefBuilder.newColumnDef(0),
                 DTColumnDefBuilder.newColumnDef(1).withOption('sType', 'date'),
                 DTColumnDefBuilder.newColumnDef(2),
-                DTColumnDefBuilder.newColumnDef(3)
+                DTColumnDefBuilder.newColumnDef(3).notSortable()
```
Comment (zhx828) [inline web/yo/app/scripts/controllers/genes.js]: What's the purpose of doing this?

File: web/yo/app/scripts/controllers/gene.js [hunk 2]
```diff
@@ -2764,7 +2777,6 @@ angular.module('oncokbApp')
                 });
                 var deferred6 = $q.defer();
                 firebase.database().ref('Meta/' + $routeParams.geneName).on('value', function (doc) {
-                    $scope.geneStautMessage = 'Last edit was made on ' + new Date(doc.val().lastModifiedAt) + ' by ' + doc.val().lastModifiedBy;
```
Comment (zhx828) [inline web/yo/app/scripts/controllers/gene.js]: No longer needed?

Comment (victoria34) [reply web/yo/app/scripts/controllers/gene.js]: No, I asked Jiaojiao and she said  this should be removed.

File: web/yo/app/scripts/services/mainutils.js [hunk 3]
```diff
@@ -528,6 +530,31 @@ angular.module('oncokbApp')
             var date = new Date();
             return date.getTime();
         }
+        function getAllUUIDs() {
+            return allUUIDs;
+        }
+        function setAllUUIDs(userName, uuids) {
```
Comment (zhx828) [inline web/yo/app/scripts/services/mainutils.js]: This function is serving different purposes. I suggest separating the functionalities.

File: web/yo/app/scripts/directives/reviewpanel.js [hunk 4]
```diff
@@ -113,8 +114,24 @@ angular.module('oncokbApp')
                     } else {
                         $scope.panelType = '';
                     }
+                    setUpdatedByAndTime();
+
+                    if ($scope.updatedBy && $scope.panelExist()) {
+                        mainUtils.pushAllUUIDs($scope.updatedBy, $scope.uuid);
```
Comment (zhx828) [inline web/yo/app/scripts/directives/reviewpanel.js]: uuid is not needed for this case. Just increase the array size.

File: web/yo/app/scripts/services/mainutils.js [hunk 5]
```diff
@@ -554,6 +581,13 @@ angular.module('oncokbApp')
             getHistoryData: getHistoryData,
             setUUIDInReview: setUUIDInReview,
             deleteUUID: deleteUUID,
-            updateMovingFlag: updateMovingFlag
+            updateMovingFlag: updateMovingFlag,
+            setAllUUIDs: setAllUUIDs,
```
Comment (zhx828) [inline web/yo/app/scripts/services/mainutils.js]: I don't suggest using `AllUUIDs`. This name does not give any information and later on would easily get confused with other concepts in the util.

**Pull request merged**